docker-compose.infogami-local.yaml

```diff
@@ -1,6 +1,6 @@
 ##
-## Helper docker-compose for doing local infogami dev. You probably want to run something like:
-## docker-compose -f docker-compose.yml -f docker-compose.override.yml -f docker-compose.infogami-local.yml up -d
+## Helper docker compose for doing local infogami dev. You probably want to run something like:
+## docker compose -f docker compose.yml -f docker compose.override.yml -f docker compose.infogami-local.yml up -d
```

Comment:
This is another line where a global replacement of `docker-compose` with `docker compose` doesn't work quite so well, as the `-f` flag to `docker compose` is going to try to use `docker`, rather than `compose.yaml`, which is what we want.

This should be `docker compose -f compose.yaml -f compose.override.yaml -f compose.infogami-local.yaml up -d`

Reply:
Yes. After ```docker-compose..``` is renamed as ```compose..``` this can be changed easily I believe. Will do it !